src/main/java/frc/robot/subsystems/swerve/SwerveModule.java

```diff
@@ -73,8 +75,8 @@ private enum SwerveModuleControlMode {
     private final Alert inDeadModeAlert;
 
     private final TelemetryTalonFX driveMotor;
-    private final TelemetryTalonFX steerMotor;
-    private final CANCoder absoluteSteerEncoder;
+    private final TelemetryCANSparkMax steerMotor;
+    private final RelativeEncoder absoluteSteerEncoder;
```

Comment:
The CANCoder is still needed as the CANCoder is the absolute encoder. You'll need another `steerEncoder` object